doc/TROUBLESHOOTING.md

```diff
@@ -18,15 +18,19 @@ player.getPlugin('spinner').disable();
 ```javascript
 // let's disable the loading animation (the 'spinner' plugin)
 export default class NoSpinner extends UIContainerPlugin {
-  get name() { return 'spinner' }
+  get name() {
+    return "spinner";
+  }
```

Comment:
We usually keep computed properties as one-liners whenever they are constant or computed values within a certain threshold. Our samples usually follow the same format.

Please refrain from reformatting samples if the changes aren't related to inconsistent white spacing or formatting.